Linux Platform and Clienttools run OK. But more error reported from Windows Clienttools build:
cmake options: -G "Visual Studio 14 2015" -D CMAKE_BUILD_TYPE=Release -DINCLUDE_EE_PLUGINS=ON -DUSE_AWS=OFF -DUSE_NATIVE_LIBRARIES=OFF -DCHECK_GIT_TAG=0 -DCLIENTTOOLS_ONLY=ON -DSIGN_DIRECTORY=c:/hpcc/sign -DEXTERNALS_DIRECTORY=c:/hpcc/platform_externals_vs2015 -DUSE_APR=OFF -DUSE_CASSANDRA=OFF -DUSE_CBLAS=OFF -DWITH_PLUGINS=OFF -DUSE_AZURE=OFF

errors: (more than 200 errors. just list top ones):
22:03:04      Creating library D:/J/workspace/CTEEW32-J24313/CE/W19-301/build/bin/Release/xmllib.lib and object D:/J/workspace/CTEEW32-J24313/CE/W19-301/build/bin/Release/xmllib.exp
22:03:04 libxslt_processor.obj : error LNK2019: unresolved external symbol _xmlInitMemory referenced in function "public: __thiscall CLibXslProcessor::CLibXslProcessor(void)" (??0CLibXslProcessor@@QAE@XZ) [D:\J\workspace\CTEEW32-J24313\CE\W19-301\build\system\xmllib\xmllib.vcxproj]
22:03:04 libxslt_processor.obj : error LNK2019: unresolved external symbol _xmlCleanupMemory referenced in function "public: virtual __thiscall CLibXslProcessor::~CLibXslProcessor(void)" (??1CLibXslProcessor@@UAE@XZ) [D:\J\workspace\CTEEW32-J24313\CE\W19-301\build\system\xmllib\xmllib.vcxproj]
22:03:04 libxslt_processor.obj : error LNK2019: unresolved external symbol _xmlFreeDoc referenced in function "public: virtual __thiscall CLibXmlSource::~CLibXmlSource(void)" (??1CLibXmlSource@@UAE@XZ) [D:\J\workspace\CTEEW32-J24313\CE\W19-301\build\system\xmllib\xmllib.vcxproj]
22:03:04 libxml_xpathprocessor.obj : error LNK2001: unresolved external symbol _xmlFreeDoc [D:\J\workspace\CTEEW32-J24313\CE\W19-301\build\system\xmllib\xmllib.vcxproj]
22:03:04 libxslt_processor.obj : error LNK2019: unresolved external symbol _xmlInitParser referenced in function "public: __thiscall CLibXslProcessor::CLibXslProcessor(void)" (??0CLibXslProcessor@@QAE@XZ) [D:\J\workspace\CTEEW32-J24313\CE\W19-301\build\system\xmllib\xmllib.vcxproj]
22:03:04 libxslt_processor.obj : error LNK2019: unresolved external symbol _xmlCleanupParser referenced in function "public: virtual __thiscall CLibXslProcessor::~CLibXslProcessor(void)" (??1CLibXslProcessor@@UAE@XZ) [D:\J\workspace\CTEEW32-J24313\CE\W19-301\build\system\xmllib\xmllib.vcxproj]
22:03:04 libxslt_processor.obj : error LNK2019: unresolved external symbol _xmlParseFile referenced in function "public: struct _xmlDoc * __thiscall CLibXmlSource::getParsedXml(void)" (?getParsedXml@CLibXmlSource@@QAEPAU_xmlDoc@@XZ) [D:\J\workspace\CTEEW32-J24313\CE\W19-

--- a/esp/esdllib/CMakeLists.txt
+++ b/esp/esdllib/CMakeLists.txt
@@ -73,7 +73,4 @@ IF (USE_LIBXSLT)
     include_directories (
          ${LIBXML2_INCLUDE_DIR}
     )
-    target_link_libraries ( esdl_svc_engine
-        ${LIBXML2_LIBRARIES}
-    )
 ENDIF()

--- a/esp/esdllib/esdl_xpath_extensions_unsupported.cpp
+++ b/esp/esdllib/esdl_xpath_extensions_unsupported.cpp
@@ -24,7 +24,7 @@
 #include "espcontext.hpp"
 #include "esdl_script.hpp"
 
-void registerEsdlXPathExtensions(IXpathContext *xpathContext, IEspContext *context, const StringArray &prefixes)
+void registerEsdlXPathExtensions(IXpathContext *xpathContext, IEsdlScriptContext *context, const StringArray &prefixes)
 {
     //nothing to register.  xpath will error on use of function attempt
 

--- a/system/xmllib/libxml_xpathprocessor.cpp
+++ b/system/xmllib/libxml_xpathprocessor.cpp
@@ -105,6 +105,7 @@ public:
 
     virtual void extractReferences(StringArray &functions, StringArray &variables) override
     {
+#ifndef _WIN32
         char *buf = nullptr;
         size_t len = 0;
 
@@ -120,6 +121,9 @@ public:
         while (line)
             line = extractFromLineGetNext(functions, variables, line);
         free (buf);
+#else
+        UNIMPLEMENTED;
+#endif
     }
 };
 static xmlXPathObjectPtr variableLookupFunc(void *data, const xmlChar *name, const xmlChar *ns_uri);

--- a/system/xmllib/xalan_processor.cpp
+++ b/system/xmllib/xalan_processor.cpp
@@ -16,6 +16,7 @@
 ############################################################################## */
 
 #include "xalan_processor.ipp"
+#include "xpathprocessor.hpp"
 #include "jencrypt.hpp"
 #include "jexcept.hpp"
 #include "xalanc/XPath/XObjectFactory.hpp"
@@ -691,4 +692,14 @@ void CXslTransform::message(StringBuffer& out, const char* in, IXslTransform* pT
 {
     CXslTransform* pTrans = dynamic_cast<CXslTransform*>(pTransform);
     pTrans->m_sMessages.append(in).append('\n');
+}
+
+extern ICompiledXpath* compileXpath(const char * xpath)
+{
+    UNIMPLEMENTED;
+}
+
+extern IXpathContext* getXpathContext(const char * xmldoc, bool strictParameterDeclaration, bool removeDocNamespaces)
+{
+    UNIMPLEMENTED;
 }

